share/new_eds/close_once.go

```diff
@@ -49,6 +49,14 @@ func (c *closeOnce) DataHash(ctx context.Context) (share.DataHash, error) {
 	return c.f.DataHash(ctx)
 }
 
+// AxisRoots returns AxisRoots of Accessor's underlying EDS.
```

Comment:
No need to duplicate the comment from the interface, especially when no other method has comment.